### PR TITLE
docs: enhance breakpoints and media query docs

### PR DESCRIPTION
**Pull request by tujoworker**
Thank you @kimroen for the nice drawings.

### CODE REVIEW

**Comment by tujoworker** [comment]
I adjusted the drawings to use the DNB font, even it had not the ∞ character. Also the colors are now from DNB. Hope thats ok for you.

**Comment by kimroen** [comment]
I don't mind at all 🙌 Looks a little small on mobile, but not catastrophic probably.

image

File: packages/dnb-design-system-portal/src/docs/uilib/layout/media-queries.mdx [hunk 1]
```diff
@@ -28,21 +30,26 @@ UX designers are using a 12 column system during their design processes.
...
+
+<InlineImg
+  src={BreakpointRanges}
+  caption="Breakpoint ranges clarification"
+/>
 
 ### Unknown sizes
 
-When dealing with breakpoints; UX often designs only for two sizes. This leads is with a unknown size in between. So check with your UXer your applications should behave for when the screen size is in between.
+When dealing with breakpoints; UX often designs only for two sizes. This leads with an unknown size in between breakpoints.
```
Comment (kimroen) [inline packages/dnb-design-system-portal/src/docs/uilib/layout/media-queries.mdx:52]: This section kind of lost its purpose after removing the part about contacting the designer to talk about the unknown designs in between the small and large sizes.

What are we trying to say here?

**Review by kimroen — APPROVED** [review]
Nice improvements 🙌

**Review by AadneRo — APPROVED** [review]

**Comment by tujoworker** [comment]
> Looks a little small on mobile, but not catastrophic probably.

Nice finding 💯 I removed the wrapper and just use the `img` tag now.

**Pull request merged**